zebra-chain/src/subtree.rs

```diff
@@ -14,7 +14,7 @@ pub const TRACKED_SUBTREE_HEIGHT: u8 = 16;
 
 /// A note commitment subtree index, used to identify a subtree in a shielded pool.
 /// Also used to count subtrees.
-#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Serialize, Deserialize)]
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Serialize, Deserialize, Default)]
```

Comment:
Optional: In general I prefer to see an explicit `0` or `zero()` method in code, rather than `default()`. It is easier for reviewers to find incorrect uses of `zero()` than `default()`, particularly in complicated nested types.

Reply:
It doesn't need it since the tuple implements `RangeBounds`, removed in https://github.com/ZcashFoundation/zebra/pull/7739/commits/668462d586cfe5a409c56b189478c374296e6b2d.